tools/topology/sof-apl-pcm512x.m4

```diff
@@ -2,6 +2,10 @@
 # Topology for generic Apollolake UP^2 with pcm512x codec and HDMI.
```

Comment:
I noticed than in the non-TDFB topology sof-apl-pcm512x.tplg I got no more DMIC capture devices. Something must have gone wrong. I though I had checked all non-TDFB topologies for no impact for .conf -files except for #comments and whitespace but apparently my check was not good. I need to re-check.

Reply:
This must have been a mistake in my testing. I tested again and the topology worked OK. Also double-checked by @juimonen so this can be resolved.